user/templates/templates.md

```diff
@@ -13,10 +13,13 @@ title: Templates
 
 In [Getting Started](/doc/getting-started/), we covered the distinction
 in Qubes OS between where you *install* your software and where you *run* your
-software. Your software is installed in [templates](/doc/glossary/#template).
-Each template shares its root filesystem (i.e., all of its programs and system
-files) with all the qubes based on it. [App qubes](/doc/glossary/#app-qube) are
-where you run your software and store your data.
+software. Software that you use in most everyday tasks, is installed within [templates](/doc/glossary/#template).
+
+When using Qubes OS, you will typically begin a task by opening an application in an [app qube](/doc/glossary/#app-qube). App qubes are built from a *template* qube (or more simply, just *a template*). As such, app qubes they _inherit_ all data from their template—from the template's OS or distro, to installed applications, to stored files—so its "[root filesystem](https://opensource.com/life/16/10/introduction-linux-filesystems)." App qubes do not, however, _persist_ their data back to the Template. 
+
+An understanding of [Inheritance and Persistence](#inheritance-and-persistence) concepts will help inform a clear understanding of this essential concept within Qubes OS. [Standalones](/doc/glossary/#standalone) are the only qubes in the Qubes OS ecosystem that neither _persist_ nor _inherit_ their filesystems to or from other qubes. 
+
+Templates are always built by developers, and are typically categorized by which OS or Linux distro each is based on. Qubes OS officially supports Debian and Fedora templates, and our community of developers has built and made available many more. When installing Qubes OS, officially supported templates may be installed. If you wish to add additional templates, you will need to download and install them.
```

Comment:
Probably better to link to the main template page instead of listing officially supported templates here. This is the sort of duplication that's likely to lead to erroneous information if we ever add to or remove from that list, so it's best to keep it in one place.

Reply:
I only just found the "Supported Versions" page, and had no idea to look for 3rd party things, there. IMHO it'd make more sense to move its content, onto this page. If you'd be comfortable with that. Then Debian and Fedora specific content, there. 

Since users don't read whole sections of documentation and will often just look at the page they feel is most relevant to an immediate need, I feel that would better support usability—while also keeping things maintainable. Thoughts?